test/corpus/cases.txt

```diff
@@ -0,0 +1,35 @@
+================================================================================
+Case examples
+================================================================================
+
+case value {
+  "A" -> True
+  _ -> False
+}
+
+case value {}
+
+--------------------------------------------------------------------------------
+
+(source_file
+  (case
+    (case_subjects
+      (identifier))
+    (case_clauses
+      (case_clause
+        (case_clause_patterns
+          (case_clause_pattern
+            (string
+              (quoted_content))))
+        (record
+          (constructor_name)))
+      (case_clause
+        (case_clause_patterns
+          (case_clause_pattern
+            (discard)))
+        (record
+          (constructor_name)))))
+  (case
+    (case_subjects
+      (identifier))
+    (case_clauses)))
```

Comment:
```suggestion
    (case_subjects
      (identifier))))
```

with the most recent changes I think the `case_clauses` node goes away

Reply:
Ignore me, I was doing that incorrectly.  Works just fine for me, sorry about that!